modules/rosa-hcp-upgrading-cli-tutorial.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * upgrading/rosa-hcp-upgrading.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-upgrading-cli_{context}"]
+= Upgrading with the ROSA CLI
+
+You can upgrade a {hcp-title} (ROSA) cluster manually by using the ROSA CLI (`rosa`). This method schedules the cluster for an immediate upgrade, if a more recent version is available.
+
+.Prerequisites
+
+* You have installed and configured the latest ROSA CLI on your installation host.
```

Comment:
Should you use `rosa` now that you've explained the abbreviation a couple of sentences ago.